api/apps/search/queries.js

```diff
@@ -5,7 +5,7 @@ export const matchAll = (term) => {
         multi_match: {
           query: term,
           type: 'best_fields',
-          fields: ['name.*', 'alternate_names.*'],
+          fields: ['name.*^4', 'alternate_names.*'],
```

Comment:
Oo what does this do?

Reply:
this boosts, `name` field over `alternate_names` in case of Museum of Modern Art, alternate name matching was causing Arab Museum of Modern Art to show up first.